dist/bin/dotc

```diff
@@ -95,8 +95,8 @@ case "$1" in
           # break out -D and -J options and add them to JAVA_OPTS as well
           # so they reach the JVM in time to do some good. The -D options
           # will be available as system properties.
```

Comment:
This comment matches the old behavior ("add them to JAVA_OPTS as well [as to the Scala options]"), could you please update accordingly? I double-checked the code, but I agree with your change.

Reply:
This code dates back to scalac script, maybe we should keep it:

```bash
  case "$1" in
    -D*)
      # pass to scala as well: otherwise we lose it sometimes when we
      # need it, e.g. communicating with a server compiler.
      java_args+=("$1")
      scala_args+=("$1")
      # respect user-supplied -Dscala.usejavacp
      case "$1" in -Dscala.usejavacp*) OVERRIDE_USEJAVACP="";; esac
      shift
      ;;
    -J*)
      # as with -D, pass to scala even though it will almost
      # never be used.
      java_args+=("${1:2}")
      scala_args+=("$1")
      shift
      ;;
```